inception-doc/src/main/resources/META-INF/asciidoc/docinfo.html

```diff
@@ -0,0 +1,75 @@
+<!-- Generate a nice TOC -->
+<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
```

Comment:
I'm all for a collapsing toc, but this should be done without having to resort to CDN hosts. The necessary scripts must be served by INCEpTION so it works even without internet connection.

Reply:
That is your code from stackoverflow.

Reply:
I know :) And the reason I never integrated it into WebAnno or INCEpTION is that so far I was  too lazy to look into packaging the JS along the application. Wicket provides JQuery but I believe it generates a dynamic URL for it so we can't just hardcode that URL into the docs.

Reply:
We can use the webjar which we have as a dependency anyways and extract the js by maven, e.g. via [this magic](https://stff.net/unpack-static-resources-from-webjar-with-maven-for-js-testing/).

Reply:
Webjars is a good idea. I adjusted build such that we use JQuery and tocify from the Webjar dependencies - but using Wicket magic instead of Maven magic :)

Reply:
If we copy the doc target folder to our website, do the paths work out then?

Reply:
I don't understand the question. My last commit already takes care of loading the JS files from our webjar dependencies. That covers the issue, doesn't it?

Reply:
The jquery paths are related to wicket. We normally upload the docs to inceptio.github.io . I do not understand how that works, as there is no wicket there, only jekyll.

Reply:
Got it. I forgot the website and considered only the case for the integrated help function in the application. I guess on the website, we could use the CDN (although we still might care to copy the JS to our website repo). We could add an ant task in inception-app-webapp/pom.xml line 602ff to replace the website-based URLs with Wicket-based URLs as the documentation is copied to its spot for being packaged as online help.